packages/contracts-bridge/contracts/TokenRegistry.sol

```diff
@@ -341,7 +341,9 @@ contract TokenRegistry is Initializable, XAppConnectionClient, ITokenRegistry {
     {
         // deploy and initialize the token contract
         _token = address(new UpgradeBeaconProxy(tokenBeacon, ""));
-        // initialize the token separately from the
+        // Initialize the token. Normally we initialize during proxy deployment
+        // but due to a proxy quirk, it's clearer to do it explicitly when
```

Comment:
what's a "proxy quirk"? Perhaps we can add a sentence or point the user somewhere

Reply:
the proxy quirk is that if the initdata is length 0, initialize is not called

Reply:
don't think we need to specify more

Reply:
for the record i think it makes more sense for a proxy to not call initialize if there's no initialize calldata passed in. what if the implementation doesn't *have* an initialize function?

Reply:
so this should really saw "due to awesome proxy design"